src/oge/emissions.py

```diff
@@ -721,13 +743,22 @@ def calculate_generator_nox_ef_per_unit_from_boiler_type(gen_fuel_allocated, yea
             ]
         )
     )
+
     if len(missing_nox_efs) > 0:
         logger.warning(
             """
             After filling with PM-fuel factors, NOx emission factors are still missing for the following boiler types.
             An emission factor of zero will be used for these boilers.
             Missing factors for FC prime movers are currently expected."""
         )
+        missing_nox_efs.insert(
```

Comment:
See above: We do not need BA codes on this check since it is not a plant-specific validation check